### PR TITLE
Format agent source code

### PR DESCRIPTION
**Pull request by KonstantinTyukalov**
Description: Just ran the "dotnet format" command, which formatted our source code
https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-format

Checklist:
- [x] No breaking changes - only code spacing changed, no any other formatting which may break something
- [x] Changes has been tested - I guess unit tests in CI should be enough

### CODE REVIEW

**Review by vmapetr — APPROVED** [review]

**Review by DmitriiBobreshev — APPROVED** [review]

**Review by LiliaSabitova — APPROVED** [review]

File: src/Agent.Listener/Configuration/ConfigurationManager.cs [hunk 1]
```diff
@@ -520,7 +520,7 @@ public async Task UnconfigureAsync(CommandSettings command)
                     IConfigurationProvider agentProvider = (extensionManager.GetExtensions<IConfigurationProvider>()).FirstOrDefault(x => x.ConfigurationProviderType == agentType);
                     ArgUtil.NotNull(agentProvider, agentType);
 
-                    bool isHostedServer = await checkIsHostedServer(agentProvider, settings, credProvider);;
+                    bool isHostedServer = await checkIsHostedServer(agentProvider, settings, credProvider); ;
```
Comment (Roman-Shchukin) [inline src/Agent.Listener/Configuration/ConfigurationManager.cs:523]: Double semicolon

File: src/Agent.Listener/DistributedTask.Pipelines/Yaml/TypeConverters/ConverterUtil.processes.cs [hunk 2]
```diff
@@ -34,7 +34,7 @@ internal static IList<ProcessResource> ReadProcessResources(IParser parser)
                 }
 
                 var resource = new ProcessResource { Type = scalar.Value };
-                resource.Name = ReadNonEmptyString(parser);;
+                resource.Name = ReadNonEmptyString(parser); ;
```
Comment (Roman-Shchukin) [inline src/Agent.Listener/DistributedTask.Pipelines/Yaml/TypeConverters/ConverterUtil.processes.cs:37]: Double semicolon

**Review by Roman-Shchukin — APPROVED** [review]

File: src/Agent.Listener/Configuration/ConfigurationManager.cs (hunk 1 above)
Comment (KonstantinTyukalov) [reply src/Agent.Listener/Configuration/ConfigurationManager.cs]: Fixed, thanks!

File: src/Agent.Listener/DistributedTask.Pipelines/Yaml/TypeConverters/ConverterUtil.processes.cs (hunk 2 above)
Comment (KonstantinTyukalov) [reply src/Agent.Listener/DistributedTask.Pipelines/Yaml/TypeConverters/ConverterUtil.processes.cs]: Fixed, thanks!

**Review by kirill-ivlev — APPROVED** [review]

**Pull request merged**